product_category_active/readme/ROADMAP.md

```diff
@@ -0,0 +1,5 @@
+- when you try to deactivate a category from the tree view by clicking
+  on the boolean_toggle widget, if a validation error pop-up appears
+  indicating that the element cannot be deactivated, then the element is
+  NOT deactivated, although the corresponding line turns gray as if it
+  had been disabled, until that tree view is refreshed.
```

Comment:
I believe this issue is now resolved. Could you review it and update the roadmap if necessary?

![image](https://github.com/user-attachments/assets/265f4794-5323-47bc-8b35-b578f8cfbe63)